modules/oe_paragraphs_document/README.md

```diff
@@ -0,0 +1,9 @@
+OpenEuropa Paragraphs Document
+=============================================
+
+Provides Document paragraph that renders media documents.
+
+#### Dependencies
+The OpenEuropa Paragraphs Document module requires the following dependencies:
+* [OpenEuropa Media](https://github.com/openeuropa/oe_media)
+* [Composite Reference](https://www.drupal.org/project/composite_reference)
```

Comment:
Is this dependency needed?

Reply:
Indeed the composite reference and media were not necessary. I've removed it.
Composite is exported on configuration as a dependency but it's not necessary since the default value is false.

Reply:
But this needs - oe_paragraphs:oe_paragraphs_media_field_storage